### PR TITLE
Fix issue with unneeded variable and only build OpenSSL if bundling

### PR DESCRIPTION
**Pull request by groeges**
Signed-off-by: Steve Groeger <GROEGES@uk.ibm.com>

### CODE REVIEW

**Comment by groeges** [comment]
jenkins compile xlinux

**Comment by groeges** [comment]
jenkins test sanity.functional xlinux

File: closed/autoconf/custom-hook.m4 [hunk 1]
```diff
@@ -417,7 +417,9 @@ AC_DEFUN([CONFIGURE_OPENSSL],
         if test -s "$OPENSSL_DIR/${LIBRARY_PREFIX}crypto${SHARED_LIBRARY_SUFFIX}" ; then
```
Comment (davehobbs) [inline closed/autoconf/custom-hook.m4]: Could we replace 417 to 423 with:

if  test “x$BUNDLE_OPENSSL" != x && ! test -s "$OPENSSL_DIR/${LIBRARY_PREFIX}crypto${SHARED_LIBRARY_SUFFIX}"; then
    BUILD_OPENSSL=yes
fi

**Comment by pshipton** [comment]
jenkins test sanity plinux

**Comment by pshipton** [comment]
Test failures are unrelated.

File: closed/autoconf/custom-hook.m4 (hunk 1 above)
Comment (groeges) [reply closed/autoconf/custom-hook.m4]: Was not able to do exactly as you stated but have tidied it up a bit.

**Comment by groeges** [comment]
jenkins test sanity xlinux, macos, plinux, aix, win

**Comment by groeges** [comment]
jenkins compile xlinux

**Comment by groeges** [comment]
jenkins test sanity xlinux,macos,plinux,aix,win

**Comment by pshipton** [comment]
Note the jsr292 failures are expected atm, and unrelated to this change.

**Comment by pshipton** [comment]
jenkins test sanity osx

**Comment by groeges** [comment]
jenkins test sanity osx

**Comment by groeges** [comment]
@pshipton @andrew-m-leonard Are we OK to get this merged now. - it seems the test failures are the jsr292 issues that Peter mentioned are known about and not related to these changes.

**Review by andrew-m-leonard — APPROVED** [review]

**Comment by andrew-m-leonard** [comment]
i'm happy
@pshipton ?

**Comment by andrew-m-leonard** [comment]
Builds look good (apart from known issue), merging

**Pull request merged**